addons/test_mail/tests/test_performance.py

```diff
@@ -311,7 +312,7 @@ def test_message_assignation_email(self):
     @warmup
     def test_message_assignation_inbox(self):
         record = self.env['mail.test.track'].create({'name': 'Test'})
-        with self.assertQueryCount(__system__=18, emp=21):
+        with self.assertQueryCount(__system__=18, emp=22):
```

Comment:
What are the extra queries? Shouldn't we expect less of them since we actually remove one relation?

Reply:
Seems solved, most of them are back to their original value.